lib/rom/repository/command_proxy.rb

```diff
@@ -11,11 +11,10 @@ class CommandProxy
 
       def initialize(command)
         @command = command
-        @root = Dry::Core::Inflector.singularize(command.name.relation).to_sym
       end
 
       def call(input)
-        command.call(root => input)
+        command.call(command.name.relation => input)
       end
```

Comment:
The reason is because the previous value was:

`Dry::Core::Inflector.singularize(command.name.relation).to_sym`

Which meant, if the command var for relation `:users` the command was being called with:

`command.call(:user => { ... })`

But the `tuple_path` in the input elevator called when executing a number of the commands was looking for `:users`. Changing it to `command.name.relation` instead meant that the commands were always called like:

`command.call(users: { ... })`

Which provided the right key for the `tuple_path` in the `InputEvaluator` to be traversed.

I imagine it would have been better to update how the `tuple_path` was constructed but I have 0 idea how to do that.